@kiva/kv-activity-feed/src/index.ts

```diff
@@ -110,16 +110,17 @@ export default class ActivityFeedService {
 	}
 
 	/**
-	 * Adds a like comment for the current user for the provided activity ID
+	 * Adds a child reaction to a parent for the current user for the provided activity ID
 	 *
-	 * @param commentId The comment ID to add a like
+	 * @param parentId The parent ID to add a child reaction
+	 * @param kind The kind of child reaction
 	 * @returns Whether the like was added successfully
 	 */
-	async addLikeToComment(commentId: string): Promise<boolean> {
+	async addChildReaction(kind: string, parentId: string): Promise<boolean> {
```

Comment:
Do we know which `kind` we want to expect for now? If so, let's restrict this further with TS. I don't know the correct strings off-hand, but something like `'comment' | 'like'`.

Reply:
Tweaky!
Yeah, we are expecting those at the moment, nice call!